packages/amplify_datastore/example/ios/unit_tests/AmplifySerializedModelUnitTests.swift

```diff
@@ -22,10 +22,21 @@ import Amplify
 class AmplifySerializedModelUnitTests: XCTestCase {
     
     let serializedModelMaps: [String: Any] = try! readJsonMap(filePath: "serialized_model_maps")
-    let flutterModelRegistration = SchemaData.flutterModelRegistration
+    let modelSchemaRegistry = SchemaData.modelSchemaRegistry
+    let customTypeSchemasRegistry = SchemaData.customTypeSchemaRegistry
+
+    override func setUpWithError() throws {
+        customTypeSchemasRegistry.addModelSchema(modelName: "Phone", modelSchema: SchemaData.PhoneSchema)
+        customTypeSchemasRegistry.addModelSchema(modelName: "Contact", modelSchema: SchemaData.ContactSchema)
+        customTypeSchemasRegistry.addModelSchema(modelName: "Address", modelSchema: SchemaData.AddressSchema)
```

Comment:
Can we test both model schemas and custom type schemas?

Reply:
My bad, this should be cleaned up. Both model and custom type schemas are included in this test suite, which are brought in by

```
    let modelSchemaRegistry = SchemaData.modelSchemaRegistry
    let customTypeSchemasRegistry = SchemaData.customTypeSchemaRegistry
```

I should've removed `setUpWithError`.